examples/lib/defaultHtmlSequencerUi.js

```diff
@@ -28,7 +28,7 @@ function DefaultHtmlSequencerUi(_sequencer, options) {
   function selectNewStepUi() {
     var m = $(addStepSel + ' select').val();
     if(!m) m = arguments[0];
-    $(addStepSel + ' .info').html(_sequencer.modulesInfo(m).description);
+    else $(addStepSel + ' .info').html(_sequencer.modulesInfo(m).description);
```

Comment:
Hi! Hm, @blurry-x-face are you sure about this? I believe line 30 is checking if it's unset. Wouldn't this `else` skip displaying the description if that is the case? Sorry if I misunderstand!

Thank you @bajajvinamr for your help, and happy to approve if you can explain and maybe I misunderstand!

Reply:
No @jywarren , this works fine because what was happening was that when no module was selected(`m===null`) it was still running `    $(addStepSel + ' .info').html(_sequencer.modulesInfo(m).description);
` hence showing error on calling moduleInfo with `null` value,  now after adding this `else` clause it will run only when there is a module selected hence no error will be displayed on the console if `m` was `null` 

I reviewed it in gitpod, it works fine there, we can also add this in moduleInfo function that if NULL is given as an argument it could return a statement like `No module selected` or something similar.

Reply:
Ok sounds great! Thanks for the clarification!